lib/cdo/create_header.rb

```diff
@@ -60,16 +63,14 @@ def self.get_project_info(key, ge_region: nil)
   end
 
   def self.get_create_dropdown_contents(options)
-    everyone_entries = %w(spritelab artist)
-
-    applab_gamelab = %w(applab gamelab)
-
     entries = options[:limit_project_types] == "true" ?
-      everyone_entries + ["minecraft_designer"] :
-      everyone_entries + applab_gamelab
+      %w(spritelab minecraft_designer) :
```

Comment:
`limit_project_types` does not seem to be set correctly in `prod`, but still updating icons and logic here. [Slack thread](https://codedotorg.slack.com/archives/CFTFD6BPV/p1747670507450109)